Context/InternalBlockchain/Repository/RevpopRepository.ts

```diff
@@ -101,6 +102,28 @@ export default class RevpopRepository implements RepositoryInterface {
         }
     }
 
+    async getIncomingContracts(start: string): Promise<Contract[]> {
+        const revpopContracts = await Apis.instance()
+            .db_api()
+            .exec("get_htlc_by_from", [this.eesAccount, start, 100])
+
+        const privateKey = PrivateKey.fromWif(this.accountPrivateKey)
+
+        const contracts = []
+        for (const contract of revpopContracts) {
+            const message = Aes.decrypt_with_checksum(
```

Comment:
Please, Please, Please, Please, handle exceptions, Please.

If the message cannot be decrypted - write error log and skip.